modules/administration-guide/pages/caching-images-for-faster-workspace-start.adoc

```diff
@@ -18,52 +18,52 @@ The {image-puller-name-short} provides the following parameters for configuratio
 |===
 |Parameter |Usage |Default
 
-|`CACHING_INTERVAL_HOURS` 
+|`CACHING_INTERVAL_HOURS`
 
 |DaemonSets health checks interval in hours
 |`"1"`
 
-|`CACHING_MEMORY_REQUEST` 
+|`CACHING_MEMORY_REQUEST`
 |The memory request for each cached image when the puller is running. See xref:defining-the-memory-parameters-for-the-image-puller.adoc[].
 |`10Mi`
 
-|`CACHING_MEMORY_LIMIT` 
+|`CACHING_MEMORY_LIMIT`
 |The memory limit for each cached image when the puller is running. See xref:defining-the-memory-parameters-for-the-image-puller.adoc[].
 |`20Mi`
 
-|`CACHING_CPU_REQUEST` 
-|The processor request for each cached image when the puller is running 
+|`CACHING_CPU_REQUEST`
+|The processor request for each cached image when the puller is running
 |`.05` or 50 millicores
 
-|`CACHING_CPU_LIMIT` 
-|The processor limit for each cached image when the puller is running 
+|`CACHING_CPU_LIMIT`
+|The processor limit for each cached image when the puller is running
 |`.2` or 200 millicores
 
-|`DAEMONSET_NAME` 
+|`DAEMONSET_NAME`
 |Name of DaemonSet to create
 |`{image-puller-deployment-name}`
 
-|`DEPLOYMENT_NAME` 
+|`DEPLOYMENT_NAME`
 |Name of the Deployment to create
 |`{image-puller-deployment-name}`
 
-|`NAMESPACE` 
+|`NAMESPACE`
 |{platforms-namespace} containing DaemonSet to create
 |`k8s-image-puller`
 
-|`IMAGES` 
+|`IMAGES`
 |Semicolon separated list of images to pull, in the format `__<name1>__=__<image1>__;__<name2>__=__<image2>__` See xref:defining-the-list-of-images-to-pull.adoc[].
 |
 
-|`NODE_SELECTOR` 
-|Node selector to apply to the Pods created by the DaemonSet 
+|`NODE_SELECTOR`
+|Node selector to apply to the Pods created by the DaemonSet
 |`'{}'`
 
-| `AFFINITY` 
-| Affinity applied to pods created by the DaemonSet 
+| `AFFINITY`
+| Affinity applied to pods created by the DaemonSet
```

Comment:
Capitalize "pod" or "Pod" consistently. Consider adopting the OpenShift guidelines for object naming. https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references